src/qunit.css

```diff
@@ -11,17 +11,17 @@
 
 /** Font Family and Sizes */
 
-#qunit-tests, #qunit-header, #qunit-banner, #qunit-testrunner-toolbar, #qunit-userAgent, #qunit-testresult {
+#qunit-tests, #qunit-header, #qunit-banner, #qunit-testrunner-toolbar, #qunit-version, #qunit-userAgent, #qunit-testresult {
```

Comment:
This looks like it can be reverted, since we're not creating a separate element anymore.